File: rest/accounts/instance-post-example-2/instance-post-example-2.js
// Download the Node helper library from twilio.com/docs/node/install
// These vars are your accountSid and authToken from twilio.com/user/account
var accountSid = 'AC1365ff479ef6502d85c27be6467a310c';
var authToken = "{{ auth_token }}";
var client = require('twilio')(accountSid, authToken);

client.accounts("AC1365ff479ef6502d85c27be6467a310c").update({
    status: "active"
}, function(err, account) {
    console.log(account.status);
});

File: rest/member/instance-post-example-2/instance-post-example-2.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;
class Example 
{
  static void Main(string[] args) 
  {
    // Find your Account Sid and Auth Token at twilio.com/user/account
    string AccountSid = "{{ account_sid }}";
    string AuthToken = "{{ auth_token }}";
    var twilio = new TwilioRestClient(AccountSid, AuthToken);

    twilio.DequeueQueueMember("QU5ef8732a3c49700934481addd5ce1659", "CA5ef8732a3c49700934481addd5ce1659", "http://demo.twilio.com/docs/voice.xml", "POST");
  }
}